### PR TITLE
removed some 'if' blocks

### PR DESCRIPTION
**Pull request by saranshgupta1995**
Did some small changes to make it run faster.

Essentially, if blocks in Python are slooowww!.
Going through the code, I saw some and could not resist suggesting a change.

### CODE REVIEW

**Comment by akien-mga** [comment]
> Essentially, if blocks in Python are slooowww!.

Do you have metrics?

While the first change is OK cosmetically (but probably doesn't impact execution speed much), the second one definitely makes code harder to read and probably doesn't increase build speed either.

**Comment by saranshgupta1995** [comment]
Don't remember where I read about it. 
And i agree that the second change does reduce the readability, and I'm unsure of how much relevance execution speed has in this case.
Feel free to suggest edits/disapprove as deemed appropriate.

**Comment by saranshgupta1995** [comment]
Reverted the second change as readability matters

File: core/SCsub [hunk 1]
```diff
@@ -15,20 +15,19 @@ txt = "0,0,0,0,0,0,0,0, 0,0,0,0,0,0,0,0, 0,0,0,0,0,0,0,0, 0,0,0,0,0,0,0,0"
...
 
         for i in range(len(e) >> 1):
-            if (i > 0):
-                txt += ","
             txts = "0x" + e[i * 2:i * 2 + 2]
             try:
                 int(txts, 16)
             except:
                 ec_valid = False
+            if(i > 0):
```
Comment (akien-mga) [inline core/SCsub]: There's a missing space after `if`. This should be moved back up to where it was, there's no reason to introduce a change in the `git blame` if it's not changing anything. Commits should then be squashed into one.

**Pull request closed without merging**